build/v1/types.go

```diff
@@ -1016,6 +1022,27 @@ type ImageChangeTrigger struct {
 	Paused bool `json:"paused,omitempty" protobuf:"varint,3,opt,name=paused"`
 }
 
+// ImageChangeTriggerStatus tracks both the latest resolved status of the ImageChangeTrigger policy if it is
+// specified in the BuildConfigSpec.Triggers struct
+type ImageChangeTriggerStatus struct {
+	// lastTriggeredImageID contains the resolved DockerImage sha from the ImageStreamTag referenced in the
+	// ImageChangeTrigger.From field when that trigger resulted in a Build being initiated.
+	LastTriggeredImageID string `json:"lastTriggeredImageID,omitempty" protobuf:"bytes,1,opt,name=lastTriggeredImageID"`
+
+	// from is whichever ImageStreamTag, either an explicitly cited ImageChangeTrigger ImageStreamTag in the spec,
+	// or the From setting in the build strategy.
+	From *corev1.ObjectReference `json:"from,omitempty" protobuf:"bytes,2,opt,name=from"`
```

Comment:
not sure why this is here?  don't we just need LastTriggeredImageID?

Reply:
for correlation ... the godoc needs to clarify that as well

Reply:
so yeah for now since we'll still set lastTriggeredImageID in the spec, you could correlate that way, but when we move beyond deprecation and remove that field, users can use the from field to correlate.

Reply:
I'd clean up the grammar a bit for clarity:

"from is the ImageStreamTag that is used as the source of the trigger. This can come from an ImageStream tag referenced in this BuildConfig's triggers, or the From image in this BuildConfig's build strategy."